### PR TITLE
added a getReviewById in reviews db

### PR DESCRIPTION
**Pull request by markangelodabu**
Just added a getReviewById to the db

### CODE REVIEW

File: db/models/reviews.js [hunk 1]
```diff
@@ -12,6 +12,19 @@ const getAllReviews = async () => {
   }
 };
 
+//get reviews by id
+const getReviewById = async (id) => {
+    try {
+        const { rows: [review] } = await client. query(`
```
Comment (lindsaynaki) [inline db/models/reviews.js:18]: on line 18,  it looks like there might be a space between 'client.' and 'query' - can you fix this please :)

**Review by lindsaynaki — CHANGES_REQUESTED** [review]
see comments

File: db/models/reviews.js (hunk 1 above)
Comment (markangelodabu) [reply db/models/reviews.js:18]: Will do, thanks Lindsay :)

**Review by yeonjupark95 — APPROVED** [review]
looks good! thanks

**Review by yeonjupark95 — APPROVED** [review]
looks good, thanks!

**Review by yeonjupark95 — APPROVED** [review]
thanks for correcting

**Review by lindsaynaki — APPROVED** [review]

**Pull request merged**